Python 3.5.3 (default, Jan 19 2017, 14:11:04) 
[GCC 6.3.0 20170124] on linux
Type "copyright", "credits" or "license()" for more information.
>>> 
========================= RESTART: /home/pi/sound.py =========================

Warning (from warnings module):
  File "/home/pi/sound.py", line 12
    GPIO.setup(GPIO_TRIGGER, GPIO.OUT)
RuntimeWarning: This channel is already in use, continuing anyway.  Use GPIO.setwarnings(False) to disable warnings.
Measured Distance = 56.4 cm
Measured Distance = 3466.1 cm
distance is more than 100,bye...
>>> 
